getting_started.rst

```diff
@@ -63,7 +63,8 @@ If you don't use conda (see above), create a virtual environment::
     Depending on your platform, the Python 3 interpreter could be invoked by
     ``python`` instead. This is the case for conda on Windows for example.
 
-Install ``aiohttp`` in the virtual environment::
+Activate the virtual environment and install ``aiohttp`` inside:
 
-    ./venv/bin/python -m pip install -U aiohttp
+    source ./venv/bin/activate
+    pip install -U aiohttp
```

Comment:
In fact, I'm not sure that it's useful to activate the venv.

I'm using more and more:

```
venv/bin/python -m pip install aiohttp
```

And it works well :-)